### PR TITLE
Add OMIM and HPO JSON API

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Extend the simple json api with endpoints for diagnoses and phenotypes.

**How to test**:
1. essentially covered in automated tests, but one can retrieve the json endpoints to a browser as well

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/phenotypes/views.py [hunk 1]
```diff
@@ -14,3 +15,18 @@ def hpo_terms():
...
     return data
+
+
+@hpo_bp.route("/api/v1/phenotypes", methods=["GET"])
+@public_endpoint
+def api_hpo_terms():
+    """Public API for HPO phenotype terms: retrieve all HPO terms
+
+    Args:
+        store(adapter.MongoAdapter):  an adapter to the scout database
```
Comment (northwestwitch) [inline scout/server/blueprints/phenotypes/views.py:26]: Did you copy this from some other function? Store is not passed to this function

File: tests/server/blueprints/phenotypes/conftest.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"""Fixtures for phenotypes"""
+
+import pytest
+
+
+@pytest.fixture
```
Comment (northwestwitch) [inline tests/server/blueprints/phenotypes/conftest.py:6]: 👍

**Review by northwestwitch — APPROVED** [review]
Looks good!

File: scout/server/blueprints/phenotypes/views.py (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/phenotypes/views.py:26]: Certainly copied, but for a previous iteration. That part moved to the controller!

**Pull request merged**